web/src/beta/features/PluginPlayground/Code/hook.ts

```
@@ -205,11 +202,41 @@ export default ({ files, resetVisualizer }: Props) => {
         }
       ]
     });
-  }, [files, resetVisualizer, setNotification]);
+
+    let schemaFieldsFromExtension = ymlJson.extensions.reduce<
+      CustomSchemaField[]
+    >((prv, cur) => {
+      if (!cur.schema || !cur.schema.groups) return prv;
+
+      return [...prv, ...cur.schema.groups.flatMap((g) => g.fields)];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)